### DISCUSSION TITLE
[TASK] consider floating UI for all overlays

### DISCUSSION THREAD

**Discussion by daKmoR** [ing-bank/lion#1570] [Ideas]
https://www.floating-ui.com/

**Comment by lpoulter** [comment]
I was going to ask why this over popper then I saw this.
> Floating UI is a project by the creator Tippy.js and co-author of Popper 2, designed to be a next-generation evolution of those libraries.

Issue related to my question [here](https://github.com/atomiks/floating-ui/issues/4)
